README.md

```diff
@@ -6,7 +6,7 @@ migration tool.
 ## What does `dbami` mean?
 
 `dbami` was initially inspired by the tool
-[`dbmate`](https://gitbub.com/amacneil/dbmate), and therefore takes inspiration
+[`dbmate`](https://github.com/amacneil/dbmate), and therefore takes inspiration
```

Comment:
Sadly, gitbub.com is already a registered domain. So much for my dreams of a VCS helper AI named Bubba.